azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -350,7 +464,7 @@ def test_connected_handler(self, stage, create_transport, mocker):
         stage.transport.on_mqtt_connected_handler()
         assert stage.previous.on_connected.call_count == 1
 
-    @pytest.mark.it("Completes an active connect op when the transport connected event fires")
+    @pytest.mark.it("Completes a pending connect op when the transport connected event fires")
     def test_completes_active_connect_op(self, stage, create_transport, callback):
         op = pipeline_ops_base.ConnectOperation(callback=callback)
         callback.reset_mock()
```

Comment:
Using a unique `mocker.MagicMock()` would prevent the need for having to do this.